src/pms/controller/_price_utils.py

```
@@ -28,13 +28,60 @@ def best_ask(signal: MarketSignal) -> float | None:
+        size = positive_float_or_none(raw_level.get("size"))
+        if price is not None and size is not None:
+            bids.append(price)
+    if not bids:
+        return None
+    return max(bids)
+
+
+def spread_bps_at_decision(signal: MarketSignal) -> int | None:
+    explicit_spread = nonnegative_float_or_none(signal.external_signal.get("spread_bps"))
+    if explicit_spread is not None:
+        return int(round(explicit_spread))
```

Ruff (0.15.12)

[warning] Value being cast to `int` is already an integer

Remove unnecessary `int` call

(RUF046)